nsist/__init__.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, appname, version, shortcuts, icon=DEFAULT_ICON,
         self.shortcuts = shortcuts
         self.icon = icon
         self.packages = packages or []
-        self.exclude = exclude or []
+        self.exclude = [os.path.normpath(p) for p in exclude] or []
```

Comment:
There's a test failure here because `exclude` is None by default. We just need to move the *or* inside the listcomp: `... for p in (exclude or [])]`.